### ISSUE TITLE
A11y_AzureToolsStorageExplorer_ToolbarControls_Usable: On applying high contrast black the high contrast theme does not get applied to the "Select All" drop down menu items.

### ISSUE THREAD

**Issue by zubinladwa7** [microsoft/AzureStorageExplorer#2609]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11YTCS; #DesktopApp; #A11yUsable; #Mac; #Benchmark; #HighContrast; #A11ySev4; #Azure Tools Storage Explorer;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: Version: 1.12.0
MacOS: Catalina 10.15.3

**Repro Steps:**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Navigate to View->Theme->Hit enter on “High Contrast- Black” control.”
7. Navigate to Toolbar controls->Select All dropdown and hit enter.
8. Verify if High Contrast-black is applied to the Select All dropdown menu items.
9. Check if High contrast-black is applied to the "Select All" dropdown. 

**​Actual:​**
On applying high contrast black ,the high contrast theme does not get applied to the  "Select All" drop down menu. Hence the menu items are seen in the same normal contrast even after applying High Contrast- black.

1. Same issue is observed for the dropdown menu items after applying High Contrast- Black theme.
2. Same issue is observed by applying Dark theme.

**Expected:**
When high contrast black is applied the  theme should get applied to all the screens even all the drop down menus. 

**User Impact:**
 Low vision users will have difficulty in viewing the menu items in high contrast black and dark theme because high contrast is not applied to list items. in bug.

**Comment by zubinladwa7** [comment]
We have uploaded repro video of this bug in [OneDrive](https://microsoft-my.sharepoint.com/:f:/p/v-subha/EvxHdg1zkN9PknF1uOxnOiQBejpxKYg9n21DydgQ_om0ew?e=WK2Zih) link with name "2609_A11y_AzureToolsStorageExplorer_ToolbarControls_HighContrast.mov" please refer the video for reference. 
Please let us know if there is any problem in accessing the link

**Comment by MRayermannMSFT** [comment]
@zubinladwa7 the link you have provided gives me a file not found error.

**Comment by MRayermannMSFT** [comment]
@zubinladwa7 based off of the bug description, all you should have to do is enable the OS color theming. That menu, and several others in the application,  get their color values from the OS.

![image](https://user-images.githubusercontent.com/21112697/92958804-ba996500-f41f-11ea-8ad8-7f4ee22d752b.png)

**Comment by MayuriNaik** [comment]
A11ySev4 bugs do not have any SLA defined and do not impact compliance. Having that said, these bugs do not require verification to be closed. Additionally, this bug does bot reproduce verified in application version: 1.16.0-dev.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
